db/external_sst_file_ingestion_job.h

```diff
@@ -99,6 +102,21 @@ class ExternalSstFileIngestionJob {
     assert(directories != nullptr);
   }
 
+  ~ExternalSstFileIngestionJob() {
+    for (const auto& c : file_ingesting_compactions_) {
+      cfd_->compaction_picker()->UnregisterCompaction(c);
+      delete c;
+    }
+    file_ingesting_compactions_.clear();
```

Comment:
I think the `clear()`s here are not needed.

Reply:
Fixed